sdk/verify.go

```diff
@@ -99,11 +99,13 @@ func (Client) Verify(expectedManifest []byte, manifestHistory [][]byte) error {
 // CoordinatorState represents the state of the Contrast Coordinator at a fixed point in time.
 type CoordinatorState struct {
 	// Manifests is a slice of manifests. It represents the manifest history of the Coordinator it was received from, sorted from oldest to newest.
-	Manifests [][]byte
+	Manifests [][]byte `json:"manifests"`
 	// Policies contains all policies that have been referenced in any manifest in Manifests. Used to verify the guarantees a deployment had over its lifetime.
-	Policies [][]byte
+	Policies [][]byte `json:"policies"`
 	// PEM-encoded certificate of the deployment's root CA.
-	RootCA []byte
+	RootCA []byte `json:"root_ca"`
 	// PEM-encoded certificate of the deployment's mesh CA.
-	MeshCA []byte
+	MeshCA []byte `json:"mesh_ca"`
+	// RawAttestationDoc is a raw attestation report.
+	RawAttestationDoc []byte `json:"raw_attestation_doc"`
```

Comment:
This does not belong into the `CoordinatorState`, as it's specific to the application-level validation. Instead, the response object should probably be a type defined in the `verify` package, holding both the attestation as well as a `CoordinatorState`.

Reply:
Am I correct in the assumption, that in the type

```go
type Response struct {
	// RawAttestationDoc is a raw attestation report.
	RawAttestationDoc []byte `json:"raw_attestation_doc"`

	sdk.CoordinatorState
}
```

I should be able to access 
```
resp := &Response{}
resp.Manifest = manifest
```
? I thought the embedded struct's keys were promoted to the parent struct. But this doesn't seem to be working in this case (?), so I guess we'd have to use `resp.CoordinatorState.Manifests`?